src/big_int.rs

```diff
@@ -359,6 +358,25 @@ impl BigInt {
     pub fn size(&self) -> usize {
         self.0.size_in_base(8)
     }
+
+    /// Convert the given `BigInt` into the corresponding little endian byte string.
+    pub fn to_le_bytes(&self) -> Result<Vec<u8>, BigIntError> {
+        // one byte encode 256 different numbers
+        const MODULUS: u64 = 256;
+        let modulus = BigInt::from(MODULUS);
+        // we cannot know easily which size we will need
+        // => allocate 128 bytes to avoid reallocations
+        const PRE_ALLOCATION_SIZE: usize = 128;
+        let mut res = Vec::with_capacity(PRE_ALLOCATION_SIZE);
+        let mut n = self.clone();
+        while !n.is_zero() {
+            let rem = n.reduce_ui(MODULUS)?;
+            n -= rem;
+            n = n.div_exact(&modulus)?;
+            res.push(rem as u8);
+        }
+        Ok(res)
```

Comment:
shouldn't we 'resize' the `res` vector at the end to shrink it to actual used size?

Reply:
It's the capacity that may be overestimated, so it's okay I think.